api/src/middleware/programsRouter.ts

```diff
@@ -131,4 +131,9 @@ programsRouter.put(
   }
 );
 
+programsRouter.get('/:id/certificate/:principal_id', (req, res) => {
+  const response = { behaviour: 'brings up certificate' };
```

Comment:
The API doesn't render graphical elements; I think here the more accurate behaviour would be that this route "Retrieves program completion information of a participant."